helm/provider.go

```diff
@@ -24,15 +24,29 @@ import (
 
 // Meta is the meta information structure for the provider
 type Meta struct {
-	data        *schema.ResourceData
-	Settings    *cli.EnvSettings
-	ConfigFlags *genericclioptions.ConfigFlags
-	HelmDriver  string
+	data             *schema.ResourceData
+	Settings         *cli.EnvSettings
+	KubernetesConfig KubernetesConfig
```

Comment:
I thought about these as "global values" and that all `*genericclioptions.ConfigFlags` used would inherit from these.